conf/CodeStyle/IntelliJ/update

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
shell scripts should follow the Google shell style guide: https://google.github.io/styleguide/shell.xml

I will inline my comments